src/store/modules/user/actions.ts

```diff
@@ -87,6 +107,26 @@ const actions: ActionTree<UserState, RootState> = {
   // Set User Instance Url
   setUserInstanceUrl ({ commit }, payload){
     commit(types.USER_INSTANCE_URL_UPDATED, payload)
+  },
+
+  async getFacilityLocations( { commit },payload ) {
+    let resp;
+    try{
+      resp = await UserService.getFacilityLocations(payload);
+      if(resp.status === 200 && resp.data.count > 0 && !hasError(resp)) {
+        let facilityLocations = resp.data.docs
+        facilityLocations = facilityLocations.map((location: any) => {
+          return {
+            locationSeqId: location.locationSeqId,
+            locationPath: location.areaId + location.aisleId + location.sectionId + location.levelId + location.positionId
+          }
+        })
+
+        return facilityLocations;
```

Comment:
We should also return a value when we have an error in response.

Reply:
Returned an empty array when getting any error or not getting proper response from api.